source/hardware/controllers/reloop_beatmix.rst

```diff
@@ -0,0 +1,91 @@
+Most of controls have a second function accessible through corresponding Shift buttons.
+
+-  `Manufacturer’s product page <https://www.reloop.com/reloop-beatmix>`__
+-  `Forum thread <https://mixxx.discourse.group/t/reloop-beatmix-1st-gen-mapping-implemented/28536>`__
+
+.. versionadded:: 2.4
+
+Audio
+-----
+
+This controller is a class complaint USB audio device. :term:`ASIO` driver for Windows based setups provided by manufacturer.
+
+Deck Controls
+-------------
+
+.. figure:: ../../_static/controllers/reloop_beatmix_top.png
+   :align: center
+   :width: 100%
+   :figwidth: 100%
+   :alt: Reloop Beatmix top view
+   :figclass: pretty-figures
+
+
+* 1 - **PITCH/TEMPO** is a notchless slider.
+* 2 - **FX SEL** dial. It scrolls through FX presets on corresponding FX Unit. Clicking the knob will toggle FX panel visibility in Mixxx.
+* 3 - **PARAM 2**. This one controls FX Super knob.
+* 4 - **FILTER/PARAM1**. This knob controls quick filter pot. Also it controls FX crossfader when operated with **Shift** button (T14).
+* 5 - **LOOP SIZE/MOVE**. Controls beatloop size. **Shift**-knob will adjust beatloop position. Clicking the knob will toggle samplers view.
+* 6 - **FX ON**. FX unit on/off button.
+* 7 - **BEAT MASH/LOOP IN**. Currently this button toggles Quantize mode off the corresponding deck. **Shift**-button acts as manual loop-in.
+* 8 - **AUTO LOOP/LOOP OUT**. Toggles beatloop function. **Shift**-button sets manual loop-out point.
+* 9 - **PFL** button. Toggles headphone monitoring of the corresponding deck.
+* 10 - **SCRATCH MODE**. **Attention here!** This one left unimplemented due to its strange hardware handling. If you press this button the light will toggle but nothing else is going to happen. You may use it as some sort of switching mark. Scratch mode is controlled by Shift-T22 instead.
+* 11 - **SEARCH**. Currently this button increase the jog sensitivity by 4 which makes it more suitable for searching through a loaded track. It also affects pitch control function of the jog, but not the scratch function.
+* 12 - **Jog**. It is a two zone control wheel. Top plate is touch sensitive and it will cause a deck to scratch when touched if scratch mode is on. Rim is scratch-indifferent.
+* 13 - Momentary **pitch up/down** buttons. Self explanatory i think. **Shift**-buttons will fast forward and rewind.
+* 14 - **SHIFT** button is used to access all of second layer controls. Worth noticing, there is one SHIFT button for each side which affects its own side only. Except for a navigation dial. Both shift buttons deal the same way with the navigation dial.
+* 15 - **HOTCUES**. Three hotcues are set/activated by pressing corresponding buttons and cleared by **Shift**-button.
+* 16 - **SYNC**. Toggles sync mode of the corresponding deck. **Shift**-button will reverse the playback.
+* 17 - **CUE PLAY**. Button set cue and start playing on release. **Shift**-button acts as a tap tempo.
```

Comment:
there is no tempo tap (before https://github.com/mixxxdj/mixxx/pull/12104 is merged ; )
You mean BPM tap https://manual.mixxx.org/2.4/en/chapters/appendix/mixxx_controls.html#control-[ChannelN]-bpm_tap